ui/common/src/coordsColor.ts

```diff
@@ -68,7 +68,7 @@ const changeColorHandle = (): void => {
     'amazons-purple': '#9f90b0 #7d4a8d',
     'breakthroughtroyka-purple-diag': '#E5DAF0 #957AB0',
     'breakthroughtroyka-maple': '#e8ceab #bc7944',
-  };
+  }; // @TODO: add Abalone
```

Comment:
is something outstanding for here?

Reply:
this is for alternating coords colours on chess like boards, we dont need this for abalone.